pkgs/applications/networking/browsers/chromium/common.nix

```
@@ -370,7 +370,10 @@ let
       # https://github.com/chromium/chromium/blob/d36462cc9279464395aea5e65d0893d76444a296/build/config/BUILDCONFIG.gn#L17-L44
       custom_toolchain = "//build/toolchain/linux/unbundle:default";
       host_toolchain = "//build/toolchain/linux/unbundle:host";
```

Why do we keep 
```
host_toolchain = "//build/toolchain/linux/unbundle:host";
```

---

#229265 changed this from `:default` to `:host`.

And reverting this to `:default` for non-cross builds removes another ~4k build steps.


```suggestion
      host_toolchain = "//build/toolchain/linux/unbundle:default";
```